src/ct/ct_actions_edit.cc

```diff
@@ -418,7 +418,7 @@ void CtActions::special_char_insert()
     for (gunichar ch : _pCtMainWin->get_ct_config()->specialChars.item()) {
         itemStore->add_row("", "", Glib::ustring{1, ch});
     }
-    const Gtk::TreeIter treeIter = CtDialogs::choose_item_dialog(*_pCtMainWin, _("Insert a Special Character"), itemStore);
+    const Gtk::TreeIter treeIter = CtDialogs::choose_item_dialog(*_pCtMainWin, _("Insert _Special Character"), itemStore);
```

Comment:
This won't be accepted because has an underscore accelerator but it's not a menu

Reply:
Hi, Giuseppe. I realized that the " _ " sign is used to indicate the menu. But then you will need to enter additional lines to translate the names of pop-ups and tooltips. Do you agree?"

Reply:
For example "Special Character"